### ISSUE TITLE
error - array instead javascript name 

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#2664]
After adding new language on localhost I've noticed this:

 
![image](https://cloud.githubusercontent.com/assets/5429548/26580915/7f05d592-453a-11e7-9e88-fc8f483baa03.png)

After checking live site with the same code but with only one language there is no file loaded between jquery and bootstrap notify files.

To switch frontend to English didn't helped. To switch to bootstrap neither.

After comparing prefs exported via Tools,
 
![image](https://cloud.githubusercontent.com/assets/5429548/26581025/f236b3e2-453a-11e7-95b5-6904cb1b53de.png)

**Comment by Jimmi08** [comment]
In database, correct site:
```
  'e_jslib_plugin' => 
  array (
  ),
```

not working site:
```
  'e_jslib_plugin' => 'Array',
  'e_jslib_theme' => 'Array',
```

**Comment by tgtje** [comment]
Did check if same was here...  (since using diff php versions (although all local) on multilang sites  php 7.1.4 10.1.22-MariaDB
and php 5.6.12 // 5.6.26 MYSQL and php 7.0.8  10.1.13-MariaDB  all show the same (as above correct)
  'e_jslib_plugin' => 
  array (
  ),
  'e_jslib_theme' => 
  array (
  ),
question remains : (since language in use AND seems bootstrap3 ?? maybe some error there ?

**Comment by Jimmi08** [comment]
Weird is that there is no file on that place with one lang.  I added new lang to live site (minimum plugins) and no error. PHP  7.0.18-0ubuntu0.16.04.1 10.0.29-MariaDB-0ubuntu0.16.04.1 PDO enabled.

'Array' is return as value if you have some pref with 'multilan' true and you call it on code without e_LANGUAGE.  

It looks like me as on localhost it detects some legacy behaviour and it's saving prefs wrong way. 
For now I was able to find only one problem -  theme prefs - if there is no $fiels , just set (prepared for future using) $fields = array();  it saves in database as empty record, no array. (but it can be related with problem above).

With library and js_manager I am not able to find this myself.
